torcharrow/test/test_trace.py

```diff
@@ -45,6 +45,9 @@ def g(self, other):
 
 
 class TestDFNoTrace(unittest.TestCase):
+    def setUp(self):
+        Scope.default = Scope()
```

Comment:
nit: If the scope is used by the whole class. You can use `setUpClass`. https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

It can also be achieved in a separate PR.

Reply:
@ejguan : I see. The plan is  to deprecate `Scope` though: https://github.com/facebookresearch/torcharrow/pull/37

Reply:
Then, no need to worry about it.